test/src/test/java/com/distkv/pine/CacheTest.java

```diff
@@ -0,0 +1,25 @@
+package com.distkv.pine;
+
+import com.distkv.common.exception.KeyNotFoundException;
+import com.distkv.pine.api.Pine;
+import com.distkv.pine.components.cache.PineCache;
+import com.distkv.supplier.BaseTestSupplier;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class CacheTest extends BaseTestSupplier {
+
+  @Test
+  public void testCache() throws InterruptedException {
+    Pine.init(getListeningAddress());
+
+    PineCache cache = Pine.newCache((long) 5000);
+    //cache.newItem("zhangsan");
+    cache.newItems("zhangsan");
+    Assert.assertEquals(cache.getItem("nihao"),"zhangsan");
+    Thread.sleep((long)6000);
```

Comment:
We should not use Sleep here, this is uncertain. You can refer to the test case about expire in the Client module,